code/src/components/ThoughtBox.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable max-len */
+/* eslint-disable no-underscore-dangle */
+import React, { useState, useEffect } from 'react';
+import { formatDistance } from 'date-fns';
+import { API_URL } from 'utils/urls';
```

Comment:
I do not understand this part utils/urls. Is it something you googled? How does it work?